### ISSUE TITLE
[Feature request] KDE5 system tray integration

### ISSUE THREAD

**Issue by Soukyuu** [kotelnik/plasma-applet-network-monitor#7]
Would be nice to have it as an entry in the system tray as well, some other widgets offer that functionality (redshift, audio volume...)

**Comment by kotelnik** [comment]
You're right. Although it would allow only one connection at a time to be showed. I mean if you were connected to ethernet as well as wifi, only one of them could be showed. Also there is the possibility to adjust the behaviour to show these combined in one "connection". This is probably the way I'm going to pick... But maybe you have better idea in mind :)

**Comment by Soukyuu** [comment]
I usually only use one connection, so for me picking which connection will be shown would be enough. I'm not sure if it's possible, but if you could spawn one systray icon per connection, that would be optimal.

**Comment by kotelnik** [comment]
Yes, like knemo was doing. Currently I don't know how to do it so I'll go for one icon until I figure out how to spawn more.
